prov/psm2/src/psmx2.h

```diff
@@ -989,7 +991,8 @@ int	psmx2_domain_enable_ep(struct psmx2_fid_domain *domain, struct psmx2_fid_ep
 void	psmx2_trx_ctxt_free(struct psmx2_trx_ctxt *trx_ctxt, int usage_flags);
 struct	psmx2_trx_ctxt *psmx2_trx_ctxt_alloc(struct psmx2_fid_domain *domain,
 					     struct psmx2_ep_name *src_addr,
-					     int sep_ctxt_idx, int usage_flags);
+					     int sep_ctxt_idx, int usage_flags,
+					     uint8_t *uuid);
```

Comment:
psm2_uuid_t is defined as an array. Not sure if "psm2_uuid_t *" would work as expected. Just use "uint8_t *" is less confusing.

typedef uint8_t psm2_uuid_t[16];

Reply:
Verified.  Changing the pointer type to "psm2_uuid_t *" would cause warning on the call to psm2_ep_open(), which expect a "psm2_uuid_t" type input and is interpreted by the compiler as type "uint8_t *". 

```
../prov/psm2/src/psmx2_trx_ctxt.c:313:7: warning: passing argument 1 of 'psm2_ep_open' from incompatible pointer type [enabled by default]
       &trx_ctxt->psm2_ep, &trx_ctxt->psm2_epid);
       ^ 
In file included from ../prov/psm2/src/version.h:44:0,
                 from ../prov/psm2/src/psmx2.h:74,
                 from ../prov/psm2/src/psmx2_trx_ctxt.c:33:
/home/jxiong/install/opa-psm2/usr/include/psm2.h:829:1: note: expected 'const uint8_t *' but argument is of type 'uint8_t (*)[16]'
 psm2_ep_open(const psm2_uuid_t unique_job_key,
 ^